packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -249,6 +249,43 @@ const Conversation = WebexPlugin.extend({
     return activity;
   },
 
+
+  /**
+  * Gets an array of activities with an array of activity URLS
+  * @param {Array} activties
+  * @param {String} cluster
+  * @returns {Promise<Object>} Resolves with the activities
+  * TODO: add cluster functionality when clusters are ready
+  */
+  bulk_activities_fetch(activties, cluster) {
+    if (!cluster) {
+
+      const params = {
+        method: 'POST',
+        api: 'conversation',
+        resource: 'bulk_activities_fetch',
+        body: {
+          activityUrls
+        }
+      };
+
+      return this.webex.request(params)
+        .then((res) => {
+          const activitiesArr = [];
+
+          res.body.multistatus.forEach((statusData) => {
```

Comment:
hmmmm is multistatus ever not defined? -- we should probably be checking body?.multistyatus

Reply:
Not for this particular call but it depends on how defensive we should be against the server. How much do you trust the convo service to always return what they say they will.

Reply:
We should definitely check for `multistatus` to exist.

Reply:
I can do that, doesn't hurt to check if it exists

Reply:
@yjlo correct, just a `if (res.body.multistatus)`.